### ISSUE TITLE
Morepath needs a logo

### ISSUE THREAD

**Issue by webmaven** [morepath/morepath#180]
@faassen: thoughts, preferences? Are there existing logos you like that can serve as inspiration, or ones that you dislike so we know what to avoid?

With a tagline like "Super Powered Python Web Framework", the logos that come to mind are ones like Angular's, HTML5, CSS3, etc., but I am obviously very interested in what you think.

**Comment by webmaven** [comment]
ping

**Comment by faassen** [comment]
Okay, I really need to get off my behind here! Let's go with the purple hexagon, which I really like. Thank you SO much for your skill and patience!

Whether we'll put the M next to 'orepath' or keep it separate we can see once we get a web site design going. I'll move on to that topic now.

**Comment by webmaven** [comment]
You're very welcome!

**Comment by webmaven** [comment]
Ping

**Comment by faassen** [comment]
Well, I followed up here a while back:

https://github.com/morepath/morepath/issues/181

I also started to use the logo on github (in the overarching project, not yet in the specific project I noticed, if that's posible). Note though that I had some trouble creating the right dimensions for github so I cheated and it got stretched a bit. Perhaps you can help?

I also started creating the website structure proper but haven't moved with that since then:

https://github.com/morepath/morepath/issues/181

I think it would also be valuable have the logo both in source and various dimensions on github. Should that be in the Morepath source repo or its own github repo?

**Comment by webmaven** [comment]
I can help.

Having it in it's own repo would work, but I suggest that the appropriate place would be in the Morepath website, under a 'Logos and Branding' section, or something similar, then link to that section from within the docs.

**Comment by faassen** [comment]
Ping? I'd like to be able to maintain the logo in a github repo.

**Comment by webmaven** [comment]
Yeah, I've been seriously AFK for a while. Bear with me for a bit longer, and I'll be able to get moving on this again and bring the logo design to a satisfactory conclusion.

**Comment by mattmakai** [comment]
hey @faassen and @webmaven, any update on when there will be an official logo? I'd love to update the [Full Stack Python Morepath page](https://www.fullstackpython.com/morepath.html) with the logo and continue to expand it with more explanations & resources. Technically the latter isn't dependent upon the logo but I've noticed that pages really come together nicely when I've got all the bits in there to draw readers in.

**Comment by faneca** [comment]
Hi! Let me introduce myself first: I've been paying attention to this project since at least 6 months ago, probably more. Never tried to use it for anything though, I've no time at all for pet projects or for trying new technologies lately... but when I finally find the time, it's definitely going to be my framework of choice for the next "puppy" I have in mind ;-)

Going to the point of this thread, I really like the direction in which the logo is evolving, and wanted to share a couple of concepts that came to mind while looking at the last designs. I hope I'm not being too intrusive.

Below there's a quick edit I did to explain myself better, but please bear in mind that it was done in a hurry, **it's plain *horrible* in its current form** and has some noticeable mistakes: I'm not a graphic artist, just wanted to expose the ideas the latest designs brought to me; I don't really know if there's a way to implement the two proposed modifications, or even just one of them, while keeping the whole thing aesthetic.
The modifications would be: the addition of the two dots inside the hexagon (displacing the "M" to the right) and the (poorly drawn) slash at the "p".

![morepath-concept](https://cloud.githubusercontent.com/assets/1409227/22089896/7cf96e98-ddee-11e6-9449-cf89cd9a756f.png)

#### Justification for the colon (a.k.a. the two dots)
 The «M» inside the hexagon already looked really great, and the two slashes kind of show the point of the project... but I added the colon to:
1. make a logo that's more differentiated from the rest of the "ems inside something" out there,
2. evoke even more what the project is about (the pervasive `://` character combination), and
3. as a bonus, put the em near the rest of the name so the letter spacing is more natural

#### Justification for the slash at the "p"
 I like the project's name a lot. Unlike some others, the name has to do with what the project actually does. Plus, it works as a unique, indivisible word which identifies it very well (for instance, when you try to find some information about it on a search engine), but at the same time has an internal symmetric structure made of two terms of four characters each (btw none of the letters are repeated, which is pretty irrelevant but also appealing IMHO). I wanted to showcase that structure somehow but without really breaking the name in its two parts. It also works in combination with the other proposed modification to reinforce the evocation of an URL path.

What do you think?

**Comment by href** [comment]
Hey there!

You're not being intrusive at all. Welcome to the Morepath issue tracker :)

I like your idea a lot actually. I'm not all that sure about the "p", though I appreciate your reasoning. However, I love the colons. As it stands the M in the hexagon doesn't have enough character imho. With the added colons it becomes very distinctive.

Maybe someone can use your ideas and improve on the current state of the logo. Thanks!

**Comment by faassen** [comment]
I agree with the M with the colons, I think that's a clear improvement. Thank you so much for that!

I'd be happy to adopt that as the new logo. We need some kind of reusable format for that. ping @blaflamme 

Concerning the slash with the P, I agree with @href in that it doesn't work yet for me. But I'm excited about the ``:M`` bit.

**Comment by blaflamme** [comment]
Personally I think that's too much having semi-colon + M + hexagon + shadows + reflection. When getting smaller you'll loose clarity and it'll become just anything. If we're thinking just about «path» the semi-colon is not mandatory since it's tied to protocol. In the end I think you have to choose between what combination you prefer, semi-colon + M or hexagon + M, all of them is a bit too much. Also having the P-like italic to recall the // makes it hard to read IMHO, mixing up «styles» inside a word is not something efficient. For the point #3 that's true if the hexagon is not there, as soon as you add it the spacing then needs to be between objects. To make it better I think we should not mix the M logo with the name, I would personally use the M and write Morepath after, then its clearer you can use them separately without missing anything.

**Comment by href** [comment]
>In the end I think you have to choose between what combination you prefer, semi-colon + M or hexagon + M, all of them is a bit too much.

I agree that we now have an overloaded logo. Personally I would get rid of the hexagon, use `:m ` instead and then go for something minimal.

**Comment by href** [comment]
Just to give an example:

![screenshot of sketch 20-01-17 15-09-21](https://cloud.githubusercontent.com/assets/273163/22152314/8292491c-df22-11e6-90fb-c644b9bf0f04.png)

This guy even looks like he's looking at some code holding his chin:

![screenshot of sketch 20-01-17 15-10-48](https://cloud.githubusercontent.com/assets/273163/22152337/ad79b8c2-df22-11e6-86b1-c8d7143c8f96.png)

Back to work! 😁 

(this uses the awesome http://overpassfont.org)

**Comment by faneca** [comment]
I agree with you, @blaflamme, it's too bloated (I already stated my version is ugly because that's what I think, @webmaven's original design was so much better ;-). Also, looking at it a second time, I don't like the thing I did with the "p".

OTOH, taking into account your comment about small dimensions, I did a quick test and kind of like how it looks when at favicon size. Yeah, the spacing and margins are quite odd (that's something I never get right when, for instance, doing web frontend development), I displaced all the background with the shadows as a quick'n'dirty way to do the edit ('cause the original was a bitmap), there's a blurry part I didn't want to spend my time to correct, and so on... thus an expert hand is definitely needed to do it right (and having the original vectorial graphic would help, too :-DD). But I'll stand for the colon concept, it makes a very recognizable pattern even in small sizes. Give it a try ;-)

**Comment by blaflamme** [comment]
I have the original redone as vector graphics I'll see if we can make something out from `:M`

**Comment by rosepac** [comment]
Finally, it has an official logo. Or simply this aspect has been ignored ..?
I was writing a short article about python frameworks, and I would like to use the logo of Morepath.

Regards.. 😼

**Comment by henri-hulski** [comment]
I think Morepath has a logo for about 3 years now... 😉
If you don't get an answer here from @faassen try on discord.

**Comment by mattmakai** [comment]
what's the source image we can use as the official logo? It is a bit unclear where everyone finally landed on this

**Comment by faassen** [comment]
The officialest logo is in the organization's github avatar but we never really got everything solidly sorted out - i.e. source image. @webmaven might know more!

**Comment by blaflamme** [comment]
Here's the logo vectorized

[Morepath.pdf](https://github.com/morepath/morepath/files/4046447/Morepath.pdf)

**Comment by faassen** [comment]
Thanks!

**Comment by mattmakai** [comment]
awesome thank you so much!
